### ISSUE TITLE
Check for present start values

### ISSUE THREAD

**Issue by chrbertsch** [modelica-tools/FMUComplianceChecker#42]
It would be good to include a check  if start values are set according to  “If  initial= exact or approx, or causality = ′′input′′ a start value must be provided”. on p. 47 of the FMI 2.0 standard.

**Comment by filip-stenstrom** [comment]
This is already fixed. Start value is also required and checked for when:
```
causality="parameter" or
variability="constant"
```

**Comment by beutlich** [comment]
> This is already fixed.

Do you mean it is already fixed in v2.0.4 or in the next release?

**Comment by filip-stenstrom** [comment]
> > This is already fixed.
> 
> Do you mean it is already fixed in v2.0.4 or in the next release?

It's already in 2.0.4. I double confirmed with attached FMU as well (it doesn't contain any binaries, so use `-x` flag with Checker if you want to confirm).
[missing_start_values.zip](https://github.com/modelica-tools/FMUComplianceChecker/files/5021119/missing_start_values.zip)

**Comment by beutlich** [comment]
> It's already in 2.0.4

Which sounds odd to me since it was reported after release of v2.0.4. Maybe @chrbertsch can give a clue which version he used when he reported this issue.
